deltabot_Arduino/deltabot_Arduino.ino

```diff
@@ -1,27 +1,31 @@
 /*
- * deltabot_servos.pde
- * Created by Matt Greensmith, 25 May 2011
- * version 0.2
- * Last edit: 25 November 2011
+ * deltabot_Arduino.ino
+ * Latest version  0.3 by Rudi Cossovich
```

Comment:
Sorry, but renaming the file and changing pin assignments does not merit an authorship change.